drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioSimulationServiceImpl.java

```diff
@@ -86,62 +87,44 @@
         extends KieService<ScenarioSimulationModelContent>
         implements ScenarioSimulationService {
 
+    private static final String KIE_VERSION = "kie.version";
+    private static final String junitActivatorPackageName = "testscenario";
```

Comment:
Can you tell if you reject to keep the same style on purpose? Shouldn't we change junitActivatorPackageName on JUNIT_ACTIVATOR_PACKAGE_NAME